packaging/docker/custom/ubuntu.Dockerfile

```diff
@@ -0,0 +1,37 @@
+ARG GRAFANA_VERSION="latest-ubuntu"
+
+FROM grafana/grafana:${GRAFANA_VERSION}-ubuntu
+
+USER root
+
+ARG GF_INSTALL_IMAGE_RENDERER_PLUGIN="false"
+
+RUN if [ $GF_INSTALL_IMAGE_RENDERER_PLUGIN = "true" ]; then \
+    apt-get update && \
+    apt-get install -y chromium-browser && \

```

Comment:
Should you also do apt-get upgrade, since the Alpine based Dockerfile does this?

Reply:
I don't know. There's a risk that will break something? :smile: